com.regnosys.rosetta/src/com/regnosys/rosetta/Rosetta.xtext

```diff
@@ -51,35 +59,33 @@ Function:
 	('inputs:' inputs += Attribute+)?
 	('output:' output = Attribute)?
 	
-	(shortcuts    += ShortcutDeclaration ';'?)*
-	conditions    += Condition*
-	operation     = Operation?
-	postConditions+= PostCondition*
+	(shortcuts     += ShortcutDeclaration ';'?)*
+	conditions     += Condition*
+	operation       = Operation?
+	postConditions += PostCondition*
 ;
 
 ShortcutDeclaration:
 	'alias' RosettaNamed ':' Definable?  expression=RosettaCalcExpression
 ;
 
-Attribute:
-	RosettaNamed RosettaTyped card=RosettaCardinality Definable? Annotations?;
 
 Condition:
 	'condition' RosettaNamed? ':' Definable?
 		Annotations?
 		(expressions+=RosettaCalcExpression ';')+
 ;
 
-Operation: // TODO no name? annotations?
-	'assign-output' attribute = [Attribute|ValidID] ':' Definable?
-	expression = RosettaCalcExpression ';'?
-;
-
 PostCondition returns Condition: // TODO no name? annotations?
 	'post-condition' ':' Definable?
 	(expressions+=RosettaCalcExpression ';')+
 ;
 
+Operation: // TODO no name? annotations?
+	'assign-output' attribute = [Attribute|ValidID] ':' Definable?
+	expression = RosettaCalcExpression ';'?
```

Comment:
if we don't need the `;` then let's avoid them.
```suggestion
	expression = RosettaCalcExpression
```

Reply:
Okay